dd-sdk-android/src/test/kotlin/com/datadog/android/InternalProxyTest.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android
+
+import com.datadog.android.telemetry.internal.Telemetry
+import com.datadog.android.utils.forge.Configurator
+import com.nhaarman.mockitokotlin2.verify
+import fr.xgouchet.elmyr.annotation.StringForgery
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.Mockito
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.quality.Strictness
+
+@Extensions(
+    ExtendWith(MockitoExtension::class),
+    ExtendWith(ForgeExtension::class)
+)
+@MockitoSettings(strictness = Strictness.LENIENT)
+@ForgeConfiguration(Configurator::class)
+internal class InternalProxyTest {
+
+    @Test
+    fun `M proxy telemetry to RumMonitor W debug()`(
+        @StringForgery message: String
+    ) {
+        // Given
+        val mockTelemetry = Mockito.mock(Telemetry::class.java)
```

Comment:
nit: we can do `import org.mockito.Mockito.mock` and just use `mock` call directly without `Mockito.` prefix as it is done in the codebase.